### PR TITLE
Deciphering code

### PR DESCRIPTION
**Pull request by muzic194**

### CODE REVIEW

File: main.py [hunk 1]
```diff
@@ -0,0 +1,44 @@
...
+
+    results = []  # list with translations for each n i alphabet
+    frequencies = []  # list with frequencies of freuent letter in each translation
+
+    for n in range(len(alphabet)):
+        results.append(code(string, n))
+        frequencies.append((code(string, n)).count(frequent_letter))
+
+    return results[frequencies.index(
+        max(frequencies))]  # this function returns only first most frequent translation (what if there are more max?)
```
Comment (adrianwizowski) [inline main.py]: Missing blank line at the end of the file. :D

File: main.py [hunk 2]
```diff
@@ -0,0 +1,44 @@
+import operator
+
+alphabet = "abcdefghijklmnopqrstuvwxyz"  # English Alphabet
```
Comment (adrianwizowski) [inline main.py:3]: https://docs.python.org/3/library/string.html#string.ascii_lowercase

File: main.py [hunk 3]
```diff
@@ -0,0 +1,44 @@
...
+
+
+# alphabet = 'aąbcćdeęfghijklłmnńoóprsśtuwyzźż' # Polski alfabet
+
+def replace_letter(letter, n):
+    """Function takes letter_1 and returns letter_2 which is n-times on the left in alphabet"""
+
+    index = ((alphabet.find(letter.lower())) + n) % len(
+        alphabet)  # find letter in aplphabet (alphabet is variable defined above function - but can be inside)
```
Comment (adrianwizowski) [inline main.py]: That's a bad code formatting :|
https://www.python.org/dev/peps/pep-0257/

File: main.py (hunk 1 above)
Comment (adrianwizowski) [inline main.py]: What should the result for: `hzayvuhbaz` word be?

Comment (adrianwizowski) [inline main.py]: Missing tests. :P

**Review by adrianwizowski — COMMENTED** [review]
Good job. Formatting could be better (In PyCharm press Ctrl + Alt + L).
And if you are unlucky, decipher won't break anything. 🤷‍♂️

File: main.py (hunk 1 above)
Comment (muzic194) [reply main.py]: One word from this list: 
'hzayvuhbaz', 'iabzwvicba', 'jbcaxwjdcb', 'kcdbyxkedc', 'ldeczylfed', 'mefdazmgfe', 'nfgebanhgf',  'hzayvuhbaz', 'iabzwvicba', 'jbcaxwjdcb', 'kcdbyxkedc', 'ldeczylfed', 'mefdazmgfe', 'nfgebanhgf', 'oghfcboihg', 'phigdcpjih', 'qijhedqkji', 'rjkiferlkj', 'skljgfsmlk', 'tlmkhgtnml', 'umnlihuonm', 'vnomjivpon', 'wopnkjwqpo', 'xpqolkxrqp', 'yqrpmlysrq', 'hzayvuhbaz'

Comment (adrianwizowski) [reply main.py]: None of this is correct.

**Pull request merged**